### ISSUE TITLE
Explain how constraints are just models

### ISSUE THREAD

**Issue by moorepants** [moorepants/learn-multibody-dynamics#41]
There are no real constraints, as we've defined them, in real life. We only have adjacent bodies interacting with each other (equal and opposite forces/torques). The constraints are simply simplifications of this interaction. I'd like to explain this in some way, but to do so I think showing some model for force/torque interaction and then reducing it to a constraint would be needed. But maybe not. It would be nice to articulate that all these things are models (simplifications) of real phenomena.

**Comment by Peter230655** [comment]
Is it not true generally in engineering, that we simplify the real world to make it amenable to analysis and calculation?
There are no rigid bodies, no point particles, Coulomb's 'law' of friction is an approximation. There is no incompressible, zero viscose fluid, yet we use it in engineering.

**Comment by moorepants** [comment]
Yes, for sure. I now forget why I felt constraints should have some special explanation regarding this.
